modules/visualization/dev-instance/README.md

```diff
@@ -0,0 +1,85 @@
+# Dev-Intance
+
+## Description
+This module will deploy an EC2 instance with an Ubuntu Desktop reachable via NiceDCV.  It is intended for GPU compute, but can run on CPU compute (with limitations)
+
+It has installed:
+ - Ubuntu 20.04 Server Desktop
+ - NiceDCV
+ - Chrome
+ - FoxGlove
+ - Ros Noetic
+ - Gazebo
+ - RViz
+ - Jupyter
+
+It leveages AWS SecretsManager to store the username and password to acces the instance via a browser.  We dynamically generate a unique password for you.  
+
+
+The user is ALWAYS `ubuntu`.
+
+### Custom File Support
+Any file you want staged on the running instance should be saved in the `scripts/` directory of this code module.  All files placed in this directory will be copied to the running EC2 instance located at the `/home/ubuntu/scripts` directory and accessible by the `ubuntu` user.
+
+## Inputs/Outputs
+
+
+### Input Parameters
+
+
+#### Required
+- `vpc-id` - the VPC this instance will reside in --- MUST have public subnets
+
+
+#### Optional
+- `instance-type` - the type of EC2 compute to use - defaults to `g4dn.xlarge`
```

Comment:
Do we need to cover the instance count input? Since you can spin up multiple of these instances using that?

Reply:
I agree, but can we do this after reinvent?